ci/before_deploy.sh

```diff
@@ -0,0 +1,29 @@
+set -ex
+
+main() {
+    local src=$(pwd) \
+          stage=
+
+    case $TRAVIS_OS_NAME in
+        linux)
+            stage=$(mktemp -d)
+            ;;
+        osx)
+            stage=$(mktemp -d -t tmp)
+            ;;
+    esac
+
+    test -f Cargo.lock || cargo generate-lockfile
```

Comment:
Good question, that's still from trust, I can probably just remove it. I also think it'd be auto-generated otherwise anyway.